artemis-server/src/main/java/org/apache/activemq/artemis/core/config/Configuration.java

```diff
@@ -1538,4 +1538,9 @@ default String resolvePropertiesSources(String propertiesFileUrl) {
    default boolean isUsingDatabasePersistence() {
       return getStoreConfiguration() != null && getStoreConfiguration().getStoreType() == StoreConfiguration.StoreType.DATABASE;
    }
+
+   /** Timeout applied when closing sessions or connections. */
+   Configuration setCloseTimeout(int timeout);
+
+   int getCloseTimeout();
```

Comment:
`CloseTimeout` seems very generic, what about `ClientCloseTimeout` or something more specific as `SessionCloseTimeout` and `ConnectionCloseTimeout`?